src/pages/index.tsx

```diff
@@ -31,6 +32,13 @@ const data1 = [
 
 const Home = () => {
   const [t] = useTranslation()
+  const router = useRouter()
+
+  useEffect(() => {
+    if (router.query) {
+      localStorage.setItem('accessToken', router.query.accessToken as string)
+    }
+  }, [router])
```

Comment:
Let's redirect to profile page @Zt0 @DavMovsesyan2002

Reply:
Done